doc/source/cice_index.rst

```diff
@@ -28,25 +28,25 @@ either Celsius or Kelvin units).
    "a4Ds", "history field accumulations, 4D categories, vertical snow", ""
    "a4Df", "history field accumulations, 4D categories, fsd", ""
    "a_min", "minimum area concentration for computing velocity", "0.001"
-   "a_rapid_mode", ":math:`{\bullet}` brine channel diameter", ""
-   "add_mpi_barriers", ":math:`\bullet` turns on MPI barriers for communication throttling", ""
-   "advection", ":math:`\bullet` type of advection algorithm used (‘remap’ or ‘upwind’)", "remap"
+   "a_rapid_mode", "brine channel diameter", ""
+   "add_mpi_barriers", "turns on MPI barriers for communication throttling", ""
+   "advection", "type of advection algorithm used (‘remap’ or ‘upwind’)", "remap"
```

Comment:
Here and below, all the bullets are removed. From what I understand, these were used to denote namelist settings, and now that we have a separate table of namelist options, the bullets are redundant. Is that the reasoning for this change ? Maybe a mention of that rationale in the commit message would be good ?

Reply:
Interesting, I couldn't figure out why those bullets were there.  It just looked like some had them and others didn't which seemed inconsistent to me.  So I just removed them all.   If we need to restore and review all the current variables, we can do that.  Sorry about that.

Reply:
I think it's ok to delete them, and remove the mention about them at the top of the index, like you do in your latest version. Closing this conversation.

Reply:
... turns out I can't do that :p It turns out I instead hid my first comment above.